src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -247,7 +249,7 @@ public static int Main(string[] args)
             }
             catch (Exception e)
             {
-                EnsureLog(verbosity);
+                EnsureLog(GetLogLevel(verbosity));
```

Comment:
Is this call redundant? I saw it is already invoked as the first thing on the execution body? Also, can we just call once right after we retrieve the verbosity above?
I am wiring up the "push", so need to understand this part better.